### DISCUSSION TITLE
Public address expiration

### DISCUSSION THREAD

**Discussion by mohoksoft** [btcpayserver/btcpayserver#4086] [General]
Does the public address have a validity period? Expiration period?

**Comment by Ou7law007** [comment]
Bitcoin public addresses do not expire. The invoice expiring is just a feature of BTCPay Server, payments that are sent after this "expiration" are still received by you.

See: https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Invoices_CreateInvoice

**monitoringMinutes**: 

> number <minutes> Nullable The number of minutes after an invoice expired after which we are still monitoring for incoming payments. Defaults to the store's settings. (The default store settings is 1440, 1 day)

**Reply by mohoksoft** [reply]
OK, thanks.
